locations/jclouds/src/test/java/org/apache/brooklyn/location/jclouds/networking/creator/DefaultAzureArmNetworkCreatorTest.java

```diff
@@ -204,7 +199,7 @@ public void testNetworkInTemplate() {
     }
 
     @Test
-    public void testIpOptionsInTemplate() {
+    public void testIpOptionsInTemplate() throws Exception {
```

Comment:
My personal option is that it's always a good idea for test methods to declare `throws Exception`. We're not writing the tests to assert that the signature of the method throws only specific checked exceptions (or no checked exception). Therefore not including the `throws Exception` just makes it more likely that subsequent refactoring might cause tests to fail to compile (which is annoying, as it might well have nothing to do with the tests it impacts). That's my opinion from past experience. In Brooklyn, it matters a lot less because we use unchecked exceptions a lot.